lib/tasks/worldwide_taxonomy.rake

```diff
@@ -0,0 +1,26 @@
+namespace :worldwide_taxonomy do
+  task republish_world_locations: :environment do
+    DataHygiene::PublishingApiRepublisher.new(WorldLocation.all).perform
+  end
+
+  task redirect_world_location_translations_to_en: :environment do
+    base_path_prefix = "/government/world"
+    world_locations = WorldLocation.all
+
+    world_locations.each do |world_location|
+      en_slug = world_location.slug
+      destination_base_path = File.join("", base_path_prefix, en_slug)
+      content_id = world_location.content_id
+      locales = world_location.original_available_locales - [:en]
```

Comment:
Oh yeah... it isn't until the other PR gets merged. Maybe these need to be in two separate PRs 🤕